### DISCUSSION TITLE
connecting to the mail server using smtp

### DISCUSSION THREAD

**Discussion by lenny1150** [postalserver/postal#2495] [Help with using Postal]
so i installed postal  on mail.projectemailmarketing.com

now i want to use the smtp to send emails from acelle 

but everytime i want to add the server to the application this error comes up

*********************

Something went wrong! Please check your entry and try again.

Connection could not be established with host mail.projectemailmarketing.com :stream_socket_client(): unable to connect to mail.projectemailmarketing.com:25 (Connection timed out)

Configure the system to send emails through an SMTP service which can be either your own SMTP system or from a 3rd service provider.

Hostname *
mail.projectemailmarketing.com
SMTP Username *
lenmarketing-email-marketing/lenny209
SMTP Password *
••••••••••••••••••••••••
visibility
SMTP Port *
25
SMTP Encryption Method

what is mean??? 

please help anyone!

**Comment by sinnbeck** [comment]
How did you set it up as that server isn't publicly available? I don't assume you managed to get your other thread solved?

**Reply by sinnbeck** [reply]
Also ask your vps provider to open port 25. Chances are it's closed by default

**Reply by lenny1150** [reply]
@sinnbeck i got another server from contabo... thats how i got it working - okay ill ask about port 25 for the admin

**Reply by sinnbeck** [reply]
Ok so you can now access it on some other domain or?

**Reply by lenny1150** [reply]
yes - i got mail.projectemailmarketing.com:5000  -

**Reply by sinnbeck** [reply]
Ah ok. I still would suggest setting up Caddy to avoid the 5000 part and also get a certificate on there :)

Did you make sure to not use any encryption when you tried logging in? postal does not come with a certificate for sending mails. Personally I have Caddy create one for me that is compatible :)

Also did you ask your vps provider if they have opened the port? I had to explicitly ask mine to have it opened

**Reply by lenny1150** [reply]
@sinnbeck i did use the command for the caddy ... but i use the mail.projectemailmarketing.com it gives me the error : page is not working.  

what you mean encryptions ? sorry im new to this...  If you mean using ssl ... i am not . and i just checked with "Using nmap Command"  and it said it was open 

Other addresses for localhost (not scanned): ::1
Not shown: 995 closed ports
PORT     STATE SERVICE
22/tcp   open  ssh
25/tcp   open  smtp
80/tcp   open  http
443/tcp  open  https
3306/tcp open  mysql

**Reply by lenny1150** [reply]
@sinnbeck 

and this is what i get when i input the caddy output 

root@mail:~# docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy
docker: Error response from daemon: Conflict. The container name "/postal-caddy"                                                                                                                      is already in use by container "fd8aea24b72f5f6560ad485deb26d7589f98dff4ba90e62                                                                                                                     a161590f9ff993b53". You have to remove (or rename) that container to be able to                                                                                                                      reuse that name.
See 'docker run --help'.

**Reply by sinnbeck** [reply]
Ok. Weird that caddy isnt working then. Perhaps try removing it and running it again


```
docker stop fd8aea24b72f5f6560ad485deb26d7589f98dff4ba90e62
docker rm fd8aea24b72f5f6560ad485deb26d7589f98dff4ba90e62

docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy
```
And yeah I meant ssl/tls for the smtp server :)

Anyways. What is the exact error you get when trying to send using the server now ? Can you send a test mail from the `Messages->Send Message` tab ?

**Reply by lenny1150** [reply]
@sinnbeck 
so i just did that for the caddy - i dont see the difference 

root@mail:~# docker ps
CONTAINER ID   IMAGE                               COMMAND                  CREA                                                                                                                     TED              STATUS          PORTS                                                                                                                                                                                           NAMES
1351da6a404c   caddy                               "caddy run --config …"   Abou                                                                                                                     t a minute ago   Up 59 seconds                                                                                                                                                                                                   postal-caddy
7a51f73fe8ff   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   12 m                                                                                                                     inutes ago       Up 12 minutes                                                                                                                                                                                                   postal-requeuer-1
783c1ea7950c   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   12 m                                                                                                                     inutes ago       Up 12 minutes                                                                                                                                                                                                   postal-web-1
ce0278a3e900   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   12 m                                                                                                                     inutes ago       Up 12 minutes                                                                                                                                                                                                   postal-cron-1
10934afd6f8b   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   12 m                                                                                                                     inutes ago       Up 12 minutes                                                                                                                                                                                                   postal-smtp-1
aad77d7604da   ghcr.io/postalserver/postal:2.1.4   "/docker-entrypoint.…"   12 m                                                                                                                     inutes ago       Up 12 minutes                                                                                                                                                                                                   postal-worker-1
b88bbb5be739   rabbitmq:3.8                        "docker-entrypoint.s…"   15 h                                                                                                                     ours ago         Up 15 hours     4369/tcp, 5671/tcp, 15691-15692/tcp, 25672/tcp,                                                                                                                      127.0.0.1:5672->5672/tcp   postal-rabbitmq
1e991f655ac2   mariadb                             "docker-entrypoint.s…"   15 h                                                                                                                     ours ago         Up 15 hours     127.0.0.1:3306->3306/tcp                                                                                                                                                                        postal-mariadb


i was able to send an email and i got 10/10 when i tested .

the thing is me using acelle , adding this server as a smtp server - the login info does not work 

this is the error message i am getting : 

Something went wrong! Please check your entry and try again.

Connection could not be established with host mail.projectemailmarketing.com :stream_socket_client(): unable to connect to mail.projectemailmarketing.com:25 (Connection timed out) 

BTW thank you so much for your help

**Reply by sinnbeck** [reply]
Happy to help. I can connect on my CLI using `telnet mail.projectemailmarketing.com 25` so it seems that is is accessible now.

Do you have any sort of other client you can test with? Or try with telnet as shown here
https://github.com/postalserver/postal/issues/1060

**Reply by lenny1150** [reply]
@sinnbeck i do not have other client to use - but i did tried to use it on another saas that i have and did not work  - it gave me error messages  =[

**Reply by lenny1150** [reply]
@sinnbeck why can it not connect to the server?

**Reply by contactosama** [reply]
Hey lenny , how do I configure it on outlook or using credentials option available on postal portal?

Please help me I am stuck there
